### PR TITLE
Add missing mobile: methods for multiple applications management

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Now there will be a complete set of methods to test, for example, application upgrade.

### CODE REVIEW

File: lib/commands/actions.js [hunk 1]
```diff
@@ -237,6 +239,78 @@ function verifyOptions (opts) {
   return {bundleId: opts.bundleId};
 }
 
+commands.mobileInstallApp = async function (opts = {}) {
+  const {app} = opts;
+  if (_.isEmpty(app)) {
+    log.errorAndThrow(`'app' parameter is mandatory. '${app}' is given instead.`);
+  }
+
+  let dstPath = app;
```
Comment (imurchie) [inline lib/commands/actions.js]: All this logic is in [configureApp](https://github.com/appium/appium-base-driver/blob/master/lib/basedriver/helpers.js#L14), which is available in this driver as `this.helpers.configureApp`.

Comment (mykola-mokhnach) [reply lib/commands/actions.js]: I've improved it a bit to make sure all the intermediate files are cleaned up after the app is installed

Comment (imurchie) [reply lib/commands/actions.js]: I'd rather see those improvements made to the central place and not duplicate all of this over here.

Comment (mykola-mokhnach) [reply lib/commands/actions.js]: ok, point taken

**Comment by mykola-mokhnach** [comment]
@imurchie anything else to fix here?

**Pull request merged**